data/101/751/943/101751943.geojson

```diff
@@ -4,11 +4,11 @@
   "properties": {
     "edtf:cessation":"uuuu",
     "edtf:inception":"uuuu",
-    "geom:area":0.004588,
-    "geom:area_square_m":40587765.634801,
-    "geom:bbox":"23.7466838309,44.2671476015,23.8683557958,44.3631019531",
-    "geom:latitude":44.318693,
-    "geom:longitude":23.804599,
+    "geom:area":0.004994,
+    "geom:area_square_m":44184808.565606,
+    "geom:bbox":"23.748914,44.276376,23.904209,44.353732",
+    "geom:latitude":44.31775,
+    "geom:longitude":23.810509,
     "gn:population":304142,
     "iso:country":"RO",
     "lbl:latitude":44.318921,
```

Comment:
Noticing that I need to adjust a subset of label centroids in single WOF <> source matches.. I shouldn't be preserving the existing values in these cases.

Reply:
fixed in f4e5047